casepropods/family_connect_registration/plugin.py

```diff
@@ -100,20 +98,21 @@ def get_identity_registration_data(self, identity, registrations):
             })
         return items
 
-    def has_whatsapp_account(self, number):
+    def has_whatsapp_account(self, number, wait_for_response):
```

Comment:
Doesn't look like `wait_for_response` is used anywhere in the function, can probably remove it.

Reply:
Oh now I see where it is used. I still don't think there is a need for the variable, just put the "wait" string directly in your payload.